Composer/packages/ui-plugins/composer/src/defaultMenuSchema.ts

```diff
@@ -163,4 +163,12 @@ export const DefaultMenuSchema: MenuUISchema = {
     label: formatMessage('Emit a telemetry track event'),
     submenu: [SubmenuNames.Debugging],
   },
+  [SDKKinds.UpdateActivity]: {
+    label: formatMessage('Update activity'),
+    submenu: [SubmenuNames.PropertyManagement],
```

Comment:
Is this in the right submenu?

Reply:
There wasn't an exiting menu that felt great. Any ideas?